translations/management/commands/add_translations.py

```
@@ -0,0 +1,214 @@
+            action="store_true",
+            help=(
+                "Create/keep the labels as inactive (active=False). Without this flag a re-run "
+                "sets active=True, which will reactivate a previously-inactivated label."
+            ),
+        )
+
+    def _load_data(self, options):
+        try:
+            data = json.load(options["data"])
+        except json.JSONDecodeError as e:
+            raise CommandError(f"Input is not valid JSON: {e}")
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)